### ISSUE TITLE
Add App Intents background extension

### ISSUE THREAD

**Issue by keeshux** [passepartoutvpn/passepartout#610]
Rather than SiriKit:

- https://developer.apple.com/documentation/appintents
- https://developer.apple.com/documentation/appintents/making-actions-and-content-discoverable-and-widely-available

Then update Siri FAQs accordingly.

Existing templates for "Set VPN":

- Connect
- Disconnect
- Toggle
- Set on-demand on/off

**Comment by keeshux** [comment]
ProfileManager is not needed if #618 is implemented, because the profile ID will be enough to locate the existing NETunnelProviderManager.

**Comment by keeshux** [comment]
In this first step, just create intents for common automation (connect/disconnect, connect to profile).

**Comment by keeshux** [comment]
With #618, intents should be just templates for standard VPN-related Shortcuts.

**Comment by keeshux** [comment]
![Image](https://github.com/user-attachments/assets/d0a22641-c15d-4f60-87ca-55857734e9f8)

**Comment by matax87** [comment]
@keeshux have you thought of the possibility of adding custom connection actions with OTP from app intents/shortcuts in order to solve #1371? Thank you!

**Comment by keeshux** [comment]
Absolutely, it is a viable solution. This issue is quite outdated, though.
